File: src/main/java/com/optica/domain/Contacts.java
package com.optica.domain;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class Contacts {

    @Id
    @GeneratedValue
    private Long id;

    private Long userId;

    private String name;

    private String phone;


    @Override
    public String toString() {
        return name  + " # " + phone  ;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }



    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

}

File: src/main/kotlin/com/optica/domain/Contacts.kt
package com.optica.domain

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id

@Entity
class Contacts {

    @Id
    @GeneratedValue
    private val id: Long? = null

    private val name: String? = null

    private val lastName: String? = null

    private val phone: String? = null

    private val email: String? = null
}
